lib/services/azuredocdb/index.js

```diff
@@ -30,8 +30,8 @@ Handlers.provision = function(log, params, next) {
   if (!cp.allValidatorsSucceed()) {
     var reply = Reply(HttpStatus.INTERNAL_SERVER_ERROR);
     reply.value.description = 'Input validators failed.';
-    var err = new Error('Validation failed.');
-    err.code = HttpStatus.INTERNAL_SERVER_ERROR;
+    var err = new Error('Parameter validation failed.  Did you supply a DocDb parameters file?');
```

Comment:
Sorry, I didn't know "one blank after commas, two after periods" before. To me, I usually type one blank after periods, and never see two blanks in English articles.
I searched it, and found http://www.dailywritingtips.com/one-or-two-spaces-after-a-period/ for your reference.

Again, it's just a convention. You decide. :)